lang/julia/julia.py

```diff
@@ -0,0 +1,115 @@
+from talon import Context, Module, actions, settings
+
+mod = Module()
+ctx = Context()
+ctx.matches = """
+tag: user.julia
+"""
+
+ctx.lists["user.code_common_function"] = {
+    "print": "print",
+}
+
+ctx.lists["user.code_type"] = {
+    "integer": "Int",
+}
+
+
+@ctx.action_class("user")
+class UserActions:
+    def code_operator_subscript():
+        actions.user.insert_between("[", "]")
+
+    def code_insert_null():
+        actions.auto_insert("nothing")
```

Comment:
It's better to just use insert for code stuff because it doesn't go through `auto_format`.